.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: python
 python:
-  - "2.7"
```

Comment:
I'm on the fence regarding `tox-travis`. I use it for edx-drf-extensions, and we inadvertently publish an untested version of the code to PyPI. This is due to the fact that the quality build finishes before the test builds, resulting in the first build publishing to PyPI. Travis currently has no capability for post-all-build hooks, only post-each-build.